src/bin/orc-export.rs

```diff
@@ -13,30 +14,119 @@ struct Cli {
     file: PathBuf,
     /// Output file. If not provided output will be printed on console
     #[arg(short, long)]
-    output: Option<PathBuf>,
-    // TODO: head=N
-    // TODO: convert_dates
-    // TODO: format=[csv|json]
-    // TODO: columns="col1,col2"
+    output_file: Option<PathBuf>,
+    /// Output format. If not provided then the output is csv
+    #[arg(value_enum, short, long)]
+    format: Option<FileFormat>,
```

Comment:
I think clap provides a way to set a default, so wouldn't need to use option here :thinking:

Reply:
Yes, you right. Fixed